docs/policy/README.md

```diff
@@ -5,6 +5,7 @@ Nomad task groups can be configured for Autoscaling using the [scaling stanza](h
  * `source` - The APM plugin that should handle the metric query. If omitted, this defaults to using the Nomad APM. 
  * `query` - The query to run against the specified APM. Currently this query should return a single value.
  * `evaluation_interval` - Defines how often the policy is evaluated by the Autoscaler. It should be provided as a duration (i.e.: `"5s"`, `"1m"` etc). If omitted the configuration value `scan_interval` from the agent will be used.
+ * `cooldown` - A time interval after a scaling action during which no additional scaling is permitted on the resource. It should be provided as a duration (i.e.: `"5s"`, `"1m"` etc). If omitted the configuration value `policy.default_cooldown` from the agent will be used.
```

Comment:
```suggestion
 * `cooldown` - A time interval after a scaling action during which no additional scaling will be performed on the resource. It should be provided as a duration (i.e.: `"5s"`, `"1m"` etc). If omitted the configuration value `policy.default_cooldown` from the agent will be used.
```

Reply:
"will be permitted" suggests that Nomad will prevent scaling actions.

Reply:
Good shout.